### DISCUSSION TITLE
Difficulty Achieving Good Precision on Monatomic Clusters with DeepMD-Kit

### DISCUSSION THREAD

**Discussion by mcourtiel** [deepmodeling/deepmd-kit#4607] [Q&A]
Hi everyone!

I'm struggling to achieve good precision when training on monatomic clusters. I've tested this on C24 and Al45, and I consistently get poor accuracy. However, I haven't encountered this issue with water clusters or periodic systems, which makes me wonder if I'm doing something wrong.

I'm using DeepMD-Kit v2.2.10 and training on 5000 structures of Al45 over 1,000,000 epochs (with 1250 for validation).
Here’s my input file:
[input.json](https://github.com/user-attachments/files/18887084/input.json)

And here are the learning curve and test results:
![lcurve](https://github.com/user-attachments/assets/118d11a2-b380-440d-b691-9d7c9ab830a3)
![Correlation_cuvres](https://github.com/user-attachments/assets/7f77ec53-ef8f-48ac-824d-d5f509e01d2e)

I've tried several things : 
-  Tighter DFT convergence criteria on forces ;
-  Different codes (deMon2K and Orca) ;
-  Different methods (DFTB and DFT).

Unfortunately, none of these changes have led to significant improvements. 'm now considering tuning the NN hyperparameters or modifying the descriptors. Do you have any recommendations ?

Any insights would be greatly appreciated! Thanks in advance.
